Application/Frontend/components/Server/Member_List.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Avatar, Group, Text, Stack, Paper } from '@mantine/core';
+
+export function MemberList() {
+  // Hardcoded member list
+  const memberList = [
```

Comment:
Although I understand that the goal of this pull request is to focus only on the UI, this addition is just an extraction of the code from `FriendsTab.tsx`. There is nothing wrong with that as even on Discord that is also true, but what I would have preferred is a single pull request with both the front-end and back-end functionality integrated as this pull request does not add anything new in addition to the existing friend list, given that the data is hardcoded. What would differentiate this component from the `FriendsTab` component would be the functionality.